config/prod/minidream-rstudio-2023-ec2-alb.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
   - "prod/minidream-rstudio-2023-ec2.yaml"
 parameters:
   Department: "SCCE"
-  Project: "MC2 Center"
+  Project: "MC2Center"
```

Comment:
@jesusaurus pushed v7.0 in aws-infra, I'd use that and remove the 2 params.

Reply:
done